tests/test_models.py

```diff
@@ -362,15 +379,21 @@ def test_mars_local_tms():
         [-5e5, -5e5, 5e5, 5e5],
         SYRTIS_TM,
         title="Web Mercator Mars",
-        geographic_crs=MARS2000_SPHERE,
     )
     assert SYRTIS_TM == syrtis_tms.crs._pyproj_crs
+    assert syrtis_tms.geographic_crs
```

Comment:
the syrtis_tms uses a different mars ellipsoid (with a flattening) so that's why it's geographic_crs doesn't equal MARS2000_SPHERE, one could make a MARS2000_ELLIPSE CRS to do a comparison here though